contexts/ProposalContext.tsx

```diff
@@ -201,11 +200,11 @@ export function ProposalProvider({
           .sort((a, b) => (a.account.accountNum < b.account.accountNum ? 1 : -1)),
       );
     }, 1000),
-    [openbook],
+    [openbook, proposal],
   );
 
   useEffect(() => {
-    if (!orders && proposal && wallet.publicKey) {
+    if (proposal && wallet.publicKey) {
```

Comment:
Does this not execute in an infinite loop?

Reply:
good catch, wanted to push too quick. Removed the dep!